README.Rmd

```diff
@@ -53,7 +53,7 @@ e.g. `2012-01-12~` for approximately the 12th of January 2012
 - uncertain date( component)s,
 e.g. `2012-01-12?` where this data point is based on an unreliable source
 - sets of dates,
-e.g. `[2012-01-01,2012-01-12]` where the date is either 1 January 2012 or 12 January 2012 
+e.g. `{2012-01-01,2012-01-12}` where the date is either 1 January 2012 or 12 January 2012 
```

Comment:
Yes there is, [] is used to separate date and time representations from the text, punctuation marks and associated
symbols used to describe them, whereas {} is used for sets of dates - so here {} is the right annotation I think

Reply:
There is - {} indicates all date/time members within a set apply, whereas [] indicates only one date/time member within the set applies.

Reply:
So if we mean 'either/or' we should be using [] and if we mean 'all' then it should be {}. I think since we expand it to include both dates, we should keep the {} and change the wording in the README?

Reply:
There also seems to be an issue in the way we are annotating dates currently - according to the second ISO document (page 28) we are annotating dates the wrong way round.

'8.2.1 Qualification of an entire date and time expression is specified by a qualification symbol occurring at the rightmost end of the expression.
8.2.2 Qualification of a group of components is specified by a qualifier symbol occurring immediately to the
right of a time scale component, which denotes that the qualification applies to that component’s value
as well as any component values to the left of that component.
8.2.3 Qualification of an individual component is specified by a qualifier symbol (in implicit form) occurring immediately to the left of the time scale component'

Currently, we are annotating uncertainty and approximation (?/~) in the following ways:
- year only: YYYY ?/~ - MM - DD
- month only: YYYY -MM ?/~ - DD
- day only: YYYY - MM - DD ?/~ (also possible conflation with whole date uncertainty/approximation)

To be consistent with ISO, annotations need to be changed to the following:
- year only: ?/~ YYYY - MM - DD
- month only: YYYY - ?/~ MM - DD
- day only: YYYY - MM - ?/~ DD
- year-month: YYYY - MM ?/~ - DD
- whole date: YYYY - MM - DD ?/~ 
- month-day: YYYY - ?/~ MM - ?/~ DD

I will be going through the functions and their tests to change the annotations before merging the PR.

Reply:
Great catch Jael! Thank you very much for noticing this and for addressing the issue!